flight/search/segmenter.py

```diff
@@ -0,0 +1,126 @@
+"""
+Summary
+-------
+Defines the public 'segment' function that allows a polygon to be divided
+into uniform squares
+"""
+from math import ceil, sin, cos, asin
+from helper import get_bounds, AIR_DROP_AREA, calculate_dist
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+
+CELL_SIZE: float = 0.00015
+c: float = calculate_dist(AIR_DROP_AREA[0], AIR_DROP_AREA[1])
+b: float = AIR_DROP_AREA[1][0] - AIR_DROP_AREA[0][0]
+SUAS_2023_THETA: float = asin(b / c)
```

Comment:
I don't know what any of this is. Add comments. Also, `b` and `c` aren't descriptive.